uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1352,18 +1729,16 @@ sub orderModalitiesByAcq {
     }
 }
 
-################################################################
-################ getUploadIDUsingTarchiveSrcLoc#################
-################################################################
 =pod
-getUploadIDUsingTarchiveSrcLoc()
-Description:
-  - Get upload_id form the mri_upload table using tarchive SourceLocation
 
-Arguments:
-  $tarchive_srcloc: The Tarchive SourceLocation
+=head3 getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc)
+
+Gets the upload ID form the mri_upload table using tarchive's SourceLocation
+
+INPUT: tarchive's source location
```

Comment:
I always get confused when we say tarchive (and in this context it is meant to be the DICOM archive not the `tarchive` table)